torch/csrc/autograd/functions/tensor.cpp

```diff
@@ -127,9 +128,9 @@ auto Chunk::apply(const variable_list& inputs) -> variable_list {
   });
 }
 
-CopySlices::CopySlices(const Variable& base_var, TensorGeometry view, std::shared_ptr<Function> fn_)
+CopySlices::CopySlices(const Variable& base_var, TensorGeometry view_, std::shared_ptr<Function> fn_)
   : base(base_var)
-  , view(std::move(view))
+  , view(std::move(view_))
```

Comment:
nit: it's a bit misleading to use `view_` as that convention is sometimes used to name struct members (e.g. in the JIT). But it's short-range here, so it's fine.